mscore/CMakeLists.txt

```diff
@@ -87,9 +87,12 @@ if (SCRIPT_INTERFACE)
             )
 endif (SCRIPT_INTERFACE)
 
+file(GLOB MSCORE_MIXER_UI ${CMAKE_CURRENT_LIST_DIR}/mixer/*.ui )
```

Comment:
From the [CMake documentation](https://cmake.org/cmake/help/v3.17/command/file.html#glob):

> We do not recommend using GLOB to collect a list of source files from your source tree [...] the generated build system cannot know when to ask CMake to regenerate.

This is explained in more detail in https://stackoverflow.com/a/32412044

See PR #4200, which added the `mscore/widgets` directory, for a better way to do this. Relevent changes are [here](https://github.com/musescore/MuseScore/pull/4200/files?file-filters%5B%5D=.txt#diff-8cb30165e2767b8fed59e5b83daeac03), [here](https://github.com/musescore/MuseScore/pull/4200/files?file-filters%5B%5D=.txt#diff-f064411a966319ea6502fbd641d1e337R33-R34) and [here](https://github.com/musescore/MuseScore/pull/4200/files?file-filters%5B%5D=.txt#diff-f064411a966319ea6502fbd641d1e337R455).

Reply:
Thanks, in fact this may be some problem.
In another pull request I used `.cmake` and their inclusions ([here](https://github.com/musescore/MuseScore/pull/5913/files#diff-a5077eecc0c1447677b09af1731c7cbcR36))
Did this to inclusions the source code into two different projects (another project - `mtest`)  

In `.cmake` files are added along the absolute path `${CMAKE_CURRENT_LIST_DIR}/...`, therefore, I do not need to add a prefix, as you did ([here](https://github.com/musescore/MuseScore/pull/5913/files#diff-9edf6d9f3a9db1b237ea8ecf05e4c69aR2))
But my  `.cmake` uses `GLOB`, I will fix it

Reply:
Fixed for this PR